lib/proxy/util.rb

```diff
@@ -75,6 +75,10 @@ def escape_for_shell(command)
     Shellwords.escape(command)
   end
 
+  def escape_for_cmdexe(command)
+    command.split(" ")
+  end
```

Comment:
This itn't proper escaping. If you want to keep it, it should be a private method in `dns_cmd_main.rb` but we shouldn't make it a general utility.

Reply:
I think this was added here for the consistency with the method above (though I agree it's not the best place).

I did some testing on windows and realized `spawn` can't escape reliably everything even when used with splatted array. We must escape `^`, `&`, `|`, `(`, `)` with `^`. So that's should be the goal for this method and that's what should be tested.

Reply:
for the record, the problem was if the argument was `'"&'`, spawn method cuts it after escaped `"`.

Reply:
I confirm, we talked about this with Anna and I named exactly these characters Marek mentions. These must be escaped. Example: all occurences of `&` character must be escaped as `^&`. Etc.